### PR TITLE
Use network layer from RSSwiftNetworking 1.1.0

### PR DESCRIPTION
**Pull request by glm4**
#### Description:

* Migrate the existing network layer code to RSSwiftNetworking.

---

#### Notes:

* The new library still needs work, but it provides the same capabilities we previously had.

---

#### Tasks:

---

#### Risk:

* High

---

#### Preview:

* N/A

### CODE REVIEW

**Review by kstoletniy — APPROVED** [review]
👏

File: ios-base/Networking/Extensions/APIClient+Application.swift [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import RSSwiftNetworking
+
+/// Provides an easy to access APIClient implementation to use across the application
```
Comment (LetoFranco) [inline ios-base/Networking/Extensions/APIClient+Application.swift:3]:
```suggestion
/// Provides an easy access to APIClient implementation to use across the application
```

or

```suggestion
/// Provides an easy way to access APIClient implementation to use across the application
```

**Review by LetoFranco — APPROVED** [review]
Just a small comment in one of the files, but its look great to me!

File: ios-base/Networking/Extensions/APIClient+Application.swift (hunk 1 above)
Comment (glm4) [reply ios-base/Networking/Extensions/APIClient+Application.swift:3]: this is not what the sentence means. _easy to access_ means that the APIClient implementation is the one being that way. However the `to` might be unnecessary here, will change that.

**Review by pMalvasio — APPROVED** [review]

**Pull request merged**